advanced_filters/static/advanced-filters/advanced-filters.js

```diff
@@ -67,7 +67,7 @@ var OperatorHandlers = function($) {
 		// initialize select2 widget and populate field choices
 		var field = $(elm).val();
 		var choices_url = ADVANCED_FILTER_CHOICES_LOOKUP_URL + (FORM_MODEL ||
-						  MODEL_LABEL) + '/' + field;
+						  MODEL_LABEL) + '/' + field + '/';
```

Comment:
Whether your URLs require a slash or not is managed by [the Django setting `APPEND_SLASH`](https://docs.djangoproject.com/en/4.2/ref/settings/#append-slash), so I don't know if we can just change this for everyone and expect it to work... One could even [decorate a view](https://docs.djangoproject.com/en/4.2/topics/http/decorators/#common) to allow a per-view override of the behavior.

That said, we probably want to simply resolve the URL to the complete path (including fields) from Django backend and pass it to this JS script, so it's consistent with whatever path the backend has configured.

Reply:
This package configures the URL to end with a tailing slash though, so Django will always add a tailing slash to the url, because the endpoint is not configurable by the user, correct?

Reply:
Sorry, it's been a while since I did Django, I think you're likely right